tests/collectors/RequestCollectorTest.php

```
@@ -193,6 +194,58 @@ public function testCaptureLeavesRequestBodyEmptyWhenRawBodyIsEmpty(): void
+        );
+        self::assertSame(
+            SensitiveDataRedactor::PLACEHOLDER,
+            $query['token'] ?? null,
+            'Query tokens must be redacted without explicit configuration.',
+        );
+        self::assertSame(
+            SensitiveDataRedactor::PLACEHOLDER,
+            $saved['COOKIE'] ?? null,
+            'Cookie collections must be redacted without explicit configuration.',
+        );
+    }
```

PHPMD (2.15.0)

[error] testCaptureRedactsDefaultHeaderBodyAndSuperglobalSecrets accesses the super-global variable $GLOBALS. (undefined)

(Superglobals)